src/frontend/komponenter/Fagsak/Vedtak/VedtakContainer.tsx

```diff
@@ -72,13 +70,35 @@ const VedtakContainer: React.FC<IProps> = ({ behandling, fagsak }) => {
         behandling.behandlingsårsakstype ===
             Behandlingårsak.REVURDERING_FEILUTBETALT_BELØP_HELT_ELLER_DELVIS_BORTFALT;
 
-    const { hentSjekkLikhetPerioder } = useSjekkLikhetPerioder(behandling.behandlingId);
+    const { erPerioderLike, hentSjekkLikhetPerioder } = useSjekkLikhetPerioder(
+        behandling.behandlingId
+    );
+    const { angreSammenslåingAvPerioder } = useAngreSammenslåingPerioder(behandling.behandlingId);
+    const [erPerioderSammenslått, settErPerioderSammenslått] = useState<boolean>(false);
+
+    const { sammenslåPerioder, feilmelding } = useSammenslåPerioder(behandling.behandlingId);
+
+    const handleKnappTrykk = async () => {
+        const oppdaterErPerioderSammenslått = !erPerioderSammenslått;
```

Comment:
Med negeringssjekk er det lurt å hindre at bruker kan dobbeltklikke på knappen - for å unngå "koko" state mellom frontend og backend 😬

Reply:
Har lagt til `laster` state for å unngå dette.